redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt

```diff
@@ -80,6 +80,16 @@ public data class Row(
   @Default("CrossAxisAlignment.Start")
   val verticalAlignment: CrossAxisAlignment,
 
+  /**
+   * Invoked when the container scrolls. The function's `offset` is represented in units in the
+   * host's coordinate system.
+   *
+   * @see Overflow.Scroll
+   */
+  @Property(7)
+  @Default("null")
+  val onScroll: ((offset: Double) -> Unit)?,
```

Comment:
Unsure if this should be `Double` (px on Android, points on iOS) or if we should standardize by emitting `offset: Dp`.

Reply:
Yeah not sure. I suspect (hope?) most implementations will be doing a simple `> 0` or something, because any kind of synchronized scrolling isn't really going to work